### ISSUE TITLE
Click and hold when selecting an item from a menu is not working with a stylus

### ISSUE THREAD

**Issue by K4LCIFER** [xournalpp/xournalpp#3750]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: Arch Linux
 - (Linux only) Desktop environment: Gnome Wayland
 - Which version of libgtk do you use: 3.24.31
 - Version of Xournal++: 1.1.0
 - Installation method: Official release in the Arch Linux repositories.

**Describe the bug**
Clicking and holding with the stylus (tapping the stylus to the screen and keeping it on the screen without lifting it) does not work for drop down menus e.g. "File", "Edit", "View", etc. What happens is you click on the menu with the stylus, and if you then drag down to scroll through the menu items, the menu item doesn't get selected/highligted as it would if you were to click and hold and drag with a regular mouse. Furthermore, if you release the click on an item that you would want to select (as you would with a mouse by releasing the mouse button), the item does not get selected.

**To Reproduce**
1. Use a tablet stylus
2. Tap the stylus to the screen on a drop down menu, i.e. "File" to open the menu.
3. Drag the stylus down as you would to select an item, i.e. "Save"
4. Observe as you drage the stylus down that the menu items are not being highlighted as you pass over them
5. Release the click by lifting the stylus off of the screen to try and select an item, i.e. "Save",
6. Observe that the item does not get selected.

**Expected behavior**
I expect that a tablet stylus would work the same as a mouse when interacting with menus. I expect that if I were to drag the stylus down through the menus, each item that I pass over will get highlighted, and I expect that when I lift the pen on the item that I want, it will get selected the very same as it would were you to do the same operation with a mouse.

**Additional context**
The tablet that I am using is a "Wacom One".

**Comment by rolandlo** [comment]
I agree that the "click and hold" behaviour with the stylus would be convenient, As I have noticed GIMP `2.10.24` (which is Gtk 2 based) allows that behaviour whereas Gtk 3 based apps like Inkscape `1.1` and `MyPaint 2.0.1` don't. Not sure it is possible with Gtk 3 (or Gtk 4).

**Comment by K4LCIFER** [comment]
> I agree that the "click and hold" behaviour with the stylus would be convenient, As I have noticed GIMP `2.10.24` (which is Gtk 2 based) allows that behaviour whereas Gtk 3 based apps like Inkscape `1.1` and `MyPaint 2.0.1` don't. Not sure it is possible with Gtk 3 (or Gtk 4).

I have tested this with another tablet that I have (XP-Pen Artist Pro 12) and it seems that the click and hold works as expected there. Perhaps its due to how the Wacom tablets have two separate mouse pointers? It wouldn't be the first issue related to the dual wacom cursor as seen in  #3722.
